lib/publish.js

```diff
@@ -137,7 +137,12 @@ class Publish extends BaseCommand {
   publishConfigToOpts (publishConfig) {
     // create a new object that inherits from the config stack
     // then squash the css-case into camelCase opts, like we do
-    return flatten({...this.npm.config.list[0], ...publishConfig})
+    let mergedConfigs = {}
+    for (const config of this.npm.config.list)
+      mergedConfigs = {...mergedConfigs, ...config}
+    mergedConfigs = {...mergedConfigs, ...publishConfig}
+
+    return flatten(mergedConfigs)
```

Comment:
```suggestion
    return {...flatten(this.npm.config.list[0]), ...flatten(publishConfig)}
```

Reply:
ok tried this and the tests are failing 😅